src/js/components/Box/doc.js

```diff
@@ -97,14 +97,14 @@ export default Box => schema(Box, {
     ],
     pad: [
       PropTypes.oneOfType([
-        PropTypes.oneOf(['none', 'small', 'medium', 'large']),
+        PropTypes.oneOf(['none', 'xsmall', 'small', 'medium', 'large']),
```

Comment:
Why did we need to add `xsmall`?

Reply:
StyledBox used `edgeSize` for `pad` prop which has `xsmall` as a key.